numba/tests/support.py

```diff
@@ -266,6 +266,16 @@ def assertRefCount(self, *objects):
                 self.fail("Refcount changed from %d to %d for object: %r"
                           % (old, new, obj))
 
+    def assertRefCountEqual(self, *objects):
+        gc.collect()
+        rc = [sys.getrefcount(x) for x in objects]
+        for i, x in enumerate(objects[1:], start=1):
+            rc_0 = rc[0]
+            rc_i = rc[i]
+            if rc_0 != rc_i:
+                self.fail(f"Refcount for objects does not match. "
+                          f"#0({rc_0}) != #{i}({rc_i}) does not match.")
```

Comment:
```suggestion
        rc_0 = rc[0]
        for i in range(1, len(objects) - 1):
            rc_i = rc[i]
            if rc_0 != rc_i:
                self.fail(f"Refcount for objects does not match. "
                          f"#0({rc_0}) != #{i}({rc_i}) does not match.")
```
hoist loop invariant and don't think `enumerate` is needed?

Reply:
shouldn't be `len(objects) - 1`.  addressed in https://github.com/numba/numba/pull/8781/commits/70cedbc9de634ef8e7f50c8cd974fb3113a8e69e